### PR TITLE
board: frdm_k64f, frdm_kw41z: Define LED1

### PR DESCRIPTION
**Pull request by pfalcon**
To make more cross-board samples work.

Signed-off-by: Paul Sokolovsky <paul.sokolovsky@linaro.org>

### CODE REVIEW

**Comment by pfalcon** [comment]
Needed e.g. for the new sample: https://github.com/zephyrproject-rtos/zephyr/pull/4573

**Review by tyrantlizard77 — APPROVED** [review]

File: boards/arm/frdm_k64f/board.h [hunk 1]
```diff
@@ -30,11 +30,17 @@
 #define BLUE_GPIO_PIN	21
 
 /* LED0. There is no physical LED on the board with this name, so create an
- * alias to the green LED to make the basic blinky sample work.
+ * alias to the blue LED to make various samples work.
```
Comment (MaureenHelm) [inline boards/arm/frdm_k64f/board.h]: s/blue/green/

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: boards/arm/frdm_k64f/board.h (hunk 1 above)
Comment (pfalcon) [reply boards/arm/frdm_k64f/board.h]: fixed

**Comment by pfalcon** [comment]
> Will you please make the same change to frdm_kl25z as well?

Sure, done.

**Review by nashif — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**